### ISSUE TITLE
Crash when "dot" command exit code is not 0

### ISSUE THREAD

**Issue by k-markov** [erocarrera/pydot#125]
Exit code other than 0 leads to:

```
/opt/conda/lib/python3.5/site-packages/pydot.py in create(self, prog, format)
   1889                      c=p.returncode,
   1890                      out=stdout_data,
-> 1891                      err=stderr_data))
   1892         assert p.returncode == 0, p.returncode
   1893         return stdout_data

KeyError: 'cmdline'
```
which is a bug, since there is no `cmdline=... in format()` at line 1888

**Comment by johnyf** [comment]
Thanks for catching this missing argument [here](https://github.com/erocarrera/pydot/blob/b835fb1c532f73d8ae0ea4d9d18052489cdb8f93/pydot.py#L1887).

**Comment by johnyf** [comment]
Fixed by 34950be661429f754255e49ebe46f0af342d8334.
